### ISSUE TITLE
CI: JSON validation errors

### ISSUE THREAD

**Issue by glenrobson** [IIIF/api#2028]
During the new theme testing I came across the following JSON validation errors:

```
source/api/presentation/3.0/index.md: bad JSON starting at line 1026
==> Expecting property name enclosed in double quotes: line 6 column 20 (char 116)
{
  "structures": [
    {
      "id": "https://example.org/iiif/range/1",
      "type": "Range",
      "items": [ { ... } ]
    }
  ]
}

source/api/presentation/3.0/index.md: bad JSON starting at line 1058
==> Expecting property name enclosed in double quotes: line 6 column 20 (char 135)
{
  "annotations": [
    {
      "id": "https://example.org/iiif/annotationPage/1",
      "type": "AnnotationPage",
      "items": [ { ... } ]
    }
  ]
}

source/api/presentation/3.0/change-log.md ...
Found 2 bad json examples :-(
```

They were picked up by the https://github.com/IIIF/api/blob/main/scripts/check_json.py script which currently only runs with `rake ci` but will become part of the CI preview tests.

**Comment by zimeon** [comment]
I think we (I) should just fix the script to handle an ellipsis. Essentially I think we just do `s/\{\s*\.\.\.\s*\}/{}/` though I'll need to test that

**Comment by azaroth42** [comment]
Agree with changing the script to allow ellipses outside of comments

**Comment by zimeon** [comment]
Was fixed by #2029 some time ago
